2-structured-data/books/model-datastore.js

```diff
@@ -95,9 +95,9 @@ module.exports = function(config) {
       .order('title')
       .start(token);
 
-    ds.runQuery(q, function(err, entities, cursor) {
+    ds.runQuery(q, function(err, entities, nextQuery) {
       if (err) { return cb(err); }
-      var hasMore = entities.length === limit ? cursor : false;
+      var hasMore = entities.length === limit ? nextQuery.startVal : false;
```

Comment:
One thing that may work is that calling `nextQuery.start()` would return the cursor instead of setting it, but it seems awkward to overload it. Maybe just `nextQuery.getStart()`, `getEnd()`, etc.

I'll go ahead and merge this since you're cool with accessing the property directly. We can always update it later.